test/unixfs.spec.js

```diff
@@ -58,11 +57,6 @@ test("dir with 100 txt files", () => {
   assert.deepEqual(inspect(node, 1), iterate100txt)
 })
 
-test("dir with 100,000 json files", () => {
```

Comment:
I think this might be still worth having because it additionally does mapping from CHAMP to UnixFS HAMT which might have some bugs too.

Reply:
It had no effect on coverage.

...FWIW I did re-run this test, and it passed once I changed the `iterate` to be 1 indexed instead of 0 (My actual directory has JSONs from `1.json` to `100000.json` not `0.json` to `99999.json`).